### PR TITLE
Update images

### PR DESCRIPTION
**Pull request by thejoefriel**
**Issue**: #refrence the relates issues here

## What did I do?
Please provide a fine description of the changes you have,
it's recommended to use lists and checklists
to include the things you have done and things left to be done for example:

- [ ] Change th title of the document
- [ ] Style the main button

## How to test?
If possible provide us with a way to test your code maybe like checking storybook or run some tests that you have made

### CODE REVIEW

File: src/pages/presspad.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ const EarwigCaseStudy = () => {
       {isLaptop ? (
         <WhereWeCameInSection style={{ paddingTop: 0 }}>
           <Image
-            fileName="earwig-figma"
+            fileName="presspad-figma"
```
Comment (jema28) [inline src/pages/presspad.js:90]: Looks good @thejoefriel, I think we need to change the alt text here too though

**Review by jema28 — CHANGES_REQUESTED** [review]

**Comment by jema28** [comment]
@thejoefriel Can you change the base of this branch to release-v1?

**Comment by yosefanajjar** [comment]
@jema28 done

**Comment by jema28** [comment]
Been thinking about this and I think the landing page is cleaner without the photo under who we are @thejoefriel. It doesn't quite fit in the colour scheme currently but happy with the rest of the changes

**Comment by jema28** [comment]
Agreed on the image files sizes @yosefanajjar and loading times, just want to make sure they don't become pixelated/too small.

**Comment by thejoefriel** [comment]
Ok have made the changes @jema28 if you want to review and merge in.

I've left the team image in there though. At the moment there's nothing that shows us on the landing page. I think this is really important, especially since I personally feel the header image looks like a stock image. I appreciate there are different colours so I'm happy for you to edit, but we need to show people and who we are on this website otherwise it doesn't look trustworthy

**Comment by jema28** [comment]
Hey @thejoefriel, yeh I agree, what about if we use this image as the header instead? I initially used it for the story and team but I think it would work work really nicely on the landing page. What do you think? 

Screenshot 2020-11-03 at 08 42 27

**Pull request merged**